corehq/apps/data_dictionary/static/data_dictionary/js/data_dictionary.js

```diff
@@ -267,12 +267,17 @@ var propertyListItem = function (
     });
 
     self.confirmDeleteProperty = function () {
-        const $modal = $("#delete-case-prop-modal").modal('show');
-        $("#delete-case-prop-name").text(self.name);
-        $("#delete-case-prop-btn").off("click").on("click", () => {
-            self.deleted(true);
-            $modal.modal('hide');
-        });
+        const countCasesUrl = initialPageData.reverse('count_cases_for_case_type', self.caseType)
+        $.getJSON(countCasesUrl, {case_property: self.name}, function (data) {
```

Comment:
Does `getJSON` support an error callback?

Reply:
A loading spinner is also a good best practice. The easiest/laziest thing to do here is probably to add a spinner to the trigger button (`disableButton` from `hq.helpers` is a good util for this) and then remove it inside teh `getJSON` callback.

Reply:
Yeah fair. I was wondering the same thing about the error callback, and was surprised to see all usages of this in HQ appeared to just have one response handler path. I'll take another look.

Agreed that a spinner would be good, so I'll add that.